src/main/java/com/architecture_map/belarus/ArchitectureMapApplication.java

```diff
@@ -5,12 +5,12 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@OpenAPIDefinition(info = @Info(title = "Architecture map API", version = "1.0"))
-@SpringBootApplication
+//@OpenAPIDefinition(info = @Info(title = "Architecture map API", version = "1.0"))
```

Comment:
why have you commented everything here and left the class?

Reply:
I need this all, I just forgot to uncomment it out